src/LIBMONGOC_VERSION_CURRENT

```diff
@@ -1 +1 @@
-1.27.0
```

Comment:
This should have been 1.27.1, but was missed in https://github.com/mongodb/mongo-php-driver/pull/1545. That does mean our 1.19.0 and 1.19.1 releases incorrectly report a bundled version of libmongoc 1.27.0 (both releases actually include 1.27.1).

Reply:
Ah, good catch. I'll take a look into how we can ensure this information is correct, as I'm planning to use this information for our SBOM file necessary for SSDLC.